src/views/pages/Diagnostics/index.js

```diff
@@ -1,55 +1,44 @@
-import React, { useState, useEffect } from "react";
+import React, {useState, useEffect} from 'react';
 import Node from '../../../services/Node';
 
 function Diagnostics() {
+  const [data, setData] = useState({pending: undefined, diagnostics: null});
 
-    const [data, setData] = useState({ pending: undefined, diagnostics: null });
+  useEffect(() => {
+    setData({pending: true, diagnostics: {}});
 
-    useEffect(() => {
-        setData({ pending: true, diagnostics: {} });
-
-        const fetchData = async () => {
-            const node = new Node();
-            const data = await node.getDiagnosticsData();
-            return data;
-        }
-        fetchData().then((fetchedData) => {
-            setData({
-                pending: false,
-                diagnostics: fetchedData
-            })
-        }).catch(console.error);
-    }, [])
-
-    return (
-        <div> {data &&
-            <dl>
-                <dt>
-                    Genesis Hash
-                </dt>
-                <dd>
-                    {data.pending == false ? data.diagnostics[0] : ''}
-                </dd>
-                <dt>
-                    Chain
-                </dt>
-                <dd>{data.pending == false ? data.diagnostics[1] : ''}
-                </dd>
-                <dt>
-                    Node Name
-                </dt>
-                <dd>
-                    {data.pending == false ? data.diagnostics[2] : ''}
-                </dd>
-                <dt>
-                    Node Version
-                </dt>
-                <dd>
-                    {data.pending == false ? data.diagnostics[3] : ''}
-                </dd>
-            </dl>}
-        </div>);
+    const fetchData = async () => {
+      const node = new Node();
+      const data = await node.getDiagnosticsData();
+      return data;
+    };
+    fetchData()
+      .then((fetchedData) => {
+        setData({
+          pending: false,
+          diagnostics: fetchedData
+        });
+      })
+      .catch(console.error);
+  }, []);
 
+  return (
+    <div>
+      {' '}
+      {data && (
```

Comment:
the `data && ...` doesn't do anything unless you are setting `undefined` as the initial state at the top of the component